hack/undeploy-lws-operator.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+oc delete namespaces cert-manager
```

Comment:
This might be used for other components, so not sure if it is good idea to remove it.